app/controllers/objects_controller.rb

```diff
@@ -9,57 +9,69 @@ class ObjectsController < ApplicationController
   # return the PreservedObject model for the druid (supplied with druid: prefix)
   # GET /objects/:druid
   def show
-    object = PreservedObject.find_by(druid: strip_druid(params[:id]))
-    if object
-      render json: object.to_json
-    else
-      render status: 404, json: 'Object not found'
-    end
+    render json: PreservedObject.find_by!(druid: druid).to_json
```

Comment:
in general, in prescat, we've tended to bias toward `find_by!` instead of `find_by` whenever we expect the object to exist, since the former will fail fast if the object is not in the DB.